lib/FlavorDownloader.js

```diff
@@ -15,6 +15,25 @@ var util = require("util");
 var events = require("events");
 var config = require('./../common/Configuration');
 var glob = require('glob');
+var httpUtils=require('./utils/http-utils');
+
+
+function isSameM3U8(a, b) {
```

Comment:
This is a function that compare two m3u8 files?
Maybe there is "more elegant" way to check it?